app/templates/admin/sales.hbs

```diff
@@ -1 +1,18 @@
-{{outlet}}
+<div class="ui grid">
+  <div class="row">
+    <div class="{{if device.isMobile 'sixteen' 'three'}} wide column">
```

Comment:
Please see how the tabs are implemented in `events.hbs`. 
We need it like that (using the component) to make sure the tab layout changes to a different style on mobile